en/flight_modes/takeoff.md

```diff
@@ -32,21 +32,28 @@ Parameter | Description
 
 The aircraft takes off in the current direction using either *catapult/hand-launch mode* or *runway takeoff mode*.
 The mode defaults to catapult/hand launch, but can be set to runway takeoff using [RWTO_TKOFF](#RWTO_TKOFF).
+RC stick movement is ignored in both cases.
 
-In *catapult/hand launch mode* the vehicle will perform a full throttle climbout (ramp up to [RWTO_MAX_THR](#RWTO_MAX_THR) in about 2 seconds).
+### Catapult/Hand Launch {#hand_launch}
+
+In *catapult/hand launch mode* the vehicle waits to detect launch (based on acceleration trigger).
+On launch it ramps up to full throttle ([RWTO_MAX_THR](#RWTO_MAX_THR)) in about 2 seconds and then performs a full throttle climbout, with 10 degree takeoff pitch. 

```

Comment:
@hamishwillee Is will use **minimum** 10 degrees.

Reply:
Updated!